lib/src/pie/PieSliceAngularInset.tsx

```diff
@@ -2,30 +2,40 @@ import React from "react";
 import { type Color, Path, type PathProps } from "@shopify/react-native-skia";
 import { useSliceAngularInsetPath } from "./hooks/useSliceAngularInsetPath";
 import { usePieSliceContext } from "./contexts/PieSliceContext";
+import type { PathAnimationConfig } from "../hooks/useAnimatedPath";
+import { AnimatedPath } from "../cartesian/components/AnimatedPath";
 
 export type PieSliceAngularInsetData = {
   angularStrokeWidth: number;
   angularStrokeColor: Color;
 };
 
-type AdditionalPathProps = Partial<Omit<PathProps, "color" | "path">>;
+type AdditionalPathProps = Partial<Omit<PathProps, "color" | "path">> & {
+  animate?: PathAnimationConfig;
+};
 
 type PieSliceAngularInsetProps = {
   angularInset: PieSliceAngularInsetData;
 } & AdditionalPathProps;
 
 export const PieSliceAngularInset = (props: PieSliceAngularInsetProps) => {
-  const { angularInset, children, ...rest } = props;
+  const { angularInset, children, animate, ...rest } = props;
   const { slice } = usePieSliceContext();
   const [path, insetPaint] = useSliceAngularInsetPath({ slice, angularInset });
 
+  // If the path is empty, don't render anything
+  if (path.toSVGString() === "M0 0L0 0M0 0L0 0") {
```

Comment:
is this a default value of a path?

Reply:
It's the value of an empty path of the angular inset, yes. I think there's an initial render that happens when all the values haven't been set/received yet higher up so it renders just a point. It's not the best solution, but it fixes the issue.